ui/src/containers/SampleViewContainer.jsx

```diff
@@ -178,20 +168,8 @@ class SampleViewContainer extends Component {
                   inPopover
                 />
               )}
-              <MotorControls
-                save={this.props.setAttribute}
-                saveStep={setStepSize}
-                uiproperties={motorUiProperties}
-                hardwareObjects={motorhardwareObjects}
-                motorsDisabled={
-                  this.props.motorInputDisable ||
-                  this.props.queueState === QUEUE_RUNNING
-                }
-                steps={motorSteps}
-                stop={this.props.stopBeamlineAction}
-                sampleViewActions={this.props.sampleViewActions}
-                sampleViewState={this.props.sampleViewState}
```

Comment:
Accessing state lower down, where it's actually needed, helps to avoid unused props like these two.